finetune_adapter.py

```diff
@@ -34,24 +35,16 @@
 ds_config = {
     "train_micro_batch_size_per_gpu": micro_batch_size,
     "gradient_accumulation_steps": gradient_accumulation_steps,
-    "zero_optimization": {
-        "stage": 2,
-        # "overlap_comm": True,
-        # "allgather_partitions":True,
-        # "allgather_bucket_size": 2e8,
-        # "reduce_scatter":True,
-        # "contiguous_gradients": True,
-        # "round_robin_gradients": True,
-        "offload_optimizer": {"device": "cpu", "pin_memory": True},
-        "offload_param": {"device": "cpu"},
```

Comment:
offloading would require us to change the optimizer. For the 3B model with 500k trainable parameters for finetuning, this is probably not worth it.